// Copyright (c) Alex Jamieson-Binnie. All rights reserved.
// Licensed under the GPL. See License.txt in the project root for license information.

syntax = "proto3";

import "narupa/protocol/trajectory/frame.proto";
import "narupa/protocol/delimiter.proto";

package narupa.protocol.instance;

message GetFrameRequest {
}

message GetFrameResponse {
  uint32 frame_index = 1;
  oneof content {
    narupa.protocol.trajectory.FrameData frame = 2;
    narupa.protocol.Delimiter delimiter = 3;
  }
}
